Javadoc error and warnings

```
...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Cannoli_2.11 ...................................... SUCCESS [6.102s]
[INFO] Cannoli_2.11: Core ................................ FAILURE [4.372s]
[INFO] Cannoli_2.11: CLI ................................. SKIPPED
[INFO] Cannoli_2.11: Assembly ............................ SKIPPED
[INFO] Cannoli_2.11: Distribution ........................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10.612s
[INFO] Finished at: Thu Mar 22 09:01:17 PDT 2018
[INFO] Final Memory: 77M/2273M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadocs) on project cannoli-core-spark2_2.11: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.7.3,SCALAVER=2.11,SPARK_VERSION=2.2.1,label=centos/ws/core/src/main/java/org/bdgenomics/cannoli/builder/CommandBuilder.java>:287: error: @param name not found
[ERROR] * @param source mount point source and target, must not be null
[ERROR] ^
[ERROR] <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.7.3,SCALAVER=2.11,SPARK_VERSION=2.2.1,label=centos/ws/core/src/main/java/org/bdgenomics/cannoli/builder/CommandBuilder.java>:290: warning: no @param for mount
[ERROR] public final CommandBuilder addMount(final String mount) {
[ERROR] ^
[ERROR] <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.7.3,SCALAVER=2.11,SPARK_VERSION=2.2.1,label=centos/ws/core/src/main/java/org/bdgenomics/cannoli/builder/CommandBuilder.java>:313: warning: no @return
[ERROR] public final CommandBuilder addMounts(final Map<String, String> mounts) {
[ERROR] ^
[ERROR]
[ERROR] Command line was: /usr/java/jdk1.8.0_60/jre/../bin/javadoc @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in '<https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.7.3,SCALAVER=2.11,SPARK_VERSION=2.2.1,label=centos/ws/core/target/apidocs'> dir.
[ERROR] -> [Help 1]
```

--- a/core/src/main/java/org/bdgenomics/cannoli/builder/CommandBuilder.java
+++ b/core/src/main/java/org/bdgenomics/cannoli/builder/CommandBuilder.java
@@ -284,7 +284,7 @@ public abstract class CommandBuilder implements Serializable {
     /**
      * Add the specified mount point to the map of mount points for this command builder.
      *
-     * @param source mount point source and target, must not be null
+     * @param mount mount point source and target, must not be null
      * @return this command builder
      */
     public final CommandBuilder addMount(final String mount) {
@@ -309,6 +309,7 @@ public abstract class CommandBuilder implements Serializable {
      * Add the specified mount points to the map of mount points for this command builder.
      *
      * @param mounts mount points to add, must not be null
+     * @return this command builder
      */
     public final CommandBuilder addMounts(final Map<String, String> mounts) {
         checkNotNull(mounts);

